### DISCUSSION TITLE
Disable 5v5 arena or make 2s and 3s give more AP than 5s

### DISCUSSION THREAD

**Discussion by tbcxd** [chromiecraft/chromiecraft#6648] [Suggestions]
Hello 

Edit: After talking with some people another way that could fix this is to make 2v2 and 3v3 give more arena points than 5v5.

I would like to suggest to disable 5v5 arena.
The main reason being is that season 2 release is around the corner and people will use 5v5 to get the most arena points, leading to decreased activity in 3v3 and 2v2 arena.

On retail and other private servers, the activity on 5v5 was always the lowest and it was used mainly by wintraders at 3am and multiboxers as a way to gear up their toons without actually playing a real arena game.
The biggest issue I see with 5v5 on CC is that it has the potential to make the activity in 2v2 even worse than it is right now.
10 players not getting their weekly ten games in 5s, could potentially be forming 5 new teams that could queue for 2v2 instead.

There is absolutely no reason to keep it open, 
5v5 is too messy and not balanced at all, It's always the most geared people that get to 2k rating in this bracket and get their gear waaay ahead of everyone else, there's so few people who pvp right now, that it would be almost impossible to set up a team that could stand a chance against those guys.
if people want large scale pvp, they should just go wsg.

Thank you for reading.

**Comment by balleny** [comment]
I dont really see an issue with wintrading in 5s on CC so far and usually, when people start queuing 5s, its more people to chim in.
Multiboxing in pvp 5s is also a non-issue here.

Further, I could literally rewrite your whole text and exchange 2 with 5 at every place and would have the same boilerplate.
Even worse, if you look into 2s, you see utilization of imbalanced specs / class abilities literally stacked. Quite a point to argue that 5s offers more ways to counteract and balance out things.

**Reply by tbcxd** [reply]
I don't see how 5s could be balanced.

**Comment by Amrx2** [comment]
This was voted and 5s remain in game.
